content/docs/setup/kubernetes/mesh-expansion/index.md

```diff
@@ -5,272 +5,280 @@ weight: 50
+additional services to Istio's model of the mesh so that other services can find and direct traffic to them. The
+ServiceEntry contains the IP addresses, ports and labels of all VMs exposing a service.
+
+{{< text bash yaml >}}
+
+$ kubectl -n test apply -f - << EOF
+apiVersion: networking.istio.io/v1alpha3
+kind: ServiceEntry
+metadata:
+  name: vm1
+spec:
+   hosts:
+   - vm1.test.svc.cluster.local
+   ports:
+   - number: 80
+     name: http
+     protocol: HTTP
+   resolution: STATIC
+   endpoints:
+    - address: 10.128.0.17
+      ports:
+        http: 8080
+      labels:
+        app: vm1
+        version: 1
+EOF
 
-{{< text bash >}}
-$ ps aux |grep istio
-root      6941  0.0  0.2  75392 16820 ?        Ssl  21:32   0:00 /usr/local/istio/bin/node_agent --logtostderr
-root      6955  0.0  0.0  49344  3048 ?        Ss   21:32   0:00 su -s /bin/bash -c INSTANCE_IP=10.150.0.5 POD_NAME=demo-vm-1 POD_NAMESPACE=default exec /usr/local/bin/pilot-agent proxy > /var/log/istio/istio.log istio-proxy
-istio-p+  7016  0.0  0.1 215172 12096 ?        Ssl  21:32   0:00 /usr/local/bin/pilot-agent proxy
-istio-p+  7094  4.0  0.3  69540 24800 ?        Sl   21:32   0:37 /usr/local/bin/envoy -c /etc/istio/proxy/envoy-rev1.json --restart-epoch 1 --drain-time-s 2 --parent-shutdown-time-s 3 --service-cluster istio-proxy --service-node sidecar~10.150.0.5~demo-vm-1.default~default.svc.cluster.local
 {{< /text >}}
 
-Istio auth node agent is healthy:
+## Troubleshooting
 
-{{< text bash >}}
-$ sudo systemctl status istio-auth-node-agent
-● istio-auth-node-agent.service - istio-auth-node-agent: The Istio auth node agent
-   Loaded: loaded (/lib/systemd/system/istio-auth-node-agent.service; disabled; vendor preset: enabled)
-   Active: active (running) since Fri 2017-10-13 21:32:29 UTC; 9s ago
-     Docs: https://istio.io/
- Main PID: 6941 (node_agent)
-    Tasks: 5
-   Memory: 5.9M
-      CPU: 92ms
-   CGroup: /system.slice/istio-auth-node-agent.service
-           └─6941 /usr/local/istio/bin/node_agent --logtostderr
-
-Oct 13 21:32:29 demo-vm-1 systemd[1]: Started istio-auth-node-agent: The Istio auth node agent.
-Oct 13 21:32:29 demo-vm-1 node_agent[6941]: I1013 21:32:29.469314    6941 main.go:66] Starting Node Agent
-Oct 13 21:32:29 demo-vm-1 node_agent[6941]: I1013 21:32:29.469365    6941 nodeagent.go:96] Node Agent starts successfully.
-Oct 13 21:32:29 demo-vm-1 node_agent[6941]: I1013 21:32:29.483324    6941 nodeagent.go:112] Sending CSR (retrial #0) ...
-Oct 13 21:32:29 demo-vm-1 node_agent[6941]: I1013 21:32:29.862575    6941 nodeagent.go:128] CSR is approved successfully. Will renew cert in 29m59.137732603s
-{{< /text >}}
+The following are some basic trouble shooting steps for common mesh expansion issues.
 
-## Running services on a mesh expansion machine
+*    When making requests from VM to the cluster, ensure you don't run the requests as `root` or
```

Comment:
```suggestion
*    When making requests from a VM to the cluster, ensure you don't run the requests as `root` or
```

Reply:
Oh that was a typo (not sure if it was from the original text or mine). Thanks for spotting!